documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,118 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Recovery
+
+
+Please note that the steps outlined below are in continuation to the ones outlined as part of the recovery procedure described [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/table-segment-recovery.md)
```

Comment:
You are starting the document without giving any kind of context to the user. Wouldn't it be better to at least put a first sentence like: `This document describes the recovery procedure of Metadata Table Segment Attribute Index files. There are 2 Metadata Table Segments per Segment Container (Container Metadata and Storage Metadata). The recovery procedure for these 2 special Table Segments differs from the general case as we will need to update SLTS system journals (link needed) once the recovery procedure completes.`. 

With this, the reader has some context, and then you can continue with: `Please, note that the...`

Reply:
have corrected this.